docs/applied_prompting/build_chatgpt.md

```diff
@@ -0,0 +1,163 @@
+
+While the early versions of GPT-3 weren't as advanced as the current GPT-3.5 series, they were still impressive. These models have been available through an API and a <a href="https://beta.openai.com/playground">playground web UI interface</a> that lets you tune certain configuration hyperparameters and test prompts. GPT-3 gained significant traction, but it did not approach the virality of ChatGPT. 
+
+What makes ChatGPT so successful, comprared to GPT-3, is it's accessibility as a straightforward AI assistant for the average person, regardless of their knowledge of data science, language models, or AI.  
+
+In this article, I overview how chat bots like ChatGPT can be implemented using a large language model like GPT-3.
+
+## Motivation
+This article was written in part because of a tweet by <a href="https://twitter.com/goodside">Riley Goodside</a>, noting how ChatGPT could have been implemented.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">How to make your own knock-off ChatGPT using GPT‑3 (text‑davinci‑003) — where you can customize the rules to your needs, and access the resulting chatbot over an API. <a href="https://t.co/9jHrs91VHW">pic.twitter.com/9jHrs91VHW</a></p>&mdash; Riley Goodside (@goodside) <a href="https://twitter.com/goodside/status/1607487283782995968?ref_src=twsrc%5Etfw">December 26, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
+
+To summarize: ChatGPT is designed specifically for chat bots and it incorporates tuning and mitigations to make outputs "safer." Chat bots like ChatGPT can be built using a prompt that includes a transcript of previous exchanges.
+
+Both ChatGPT and models in the GPT 3.5 series use RL and human feedback to optimize outputs. ChatGPT is by no means leaps and bounds better than the core GPT models. **It largely relies on a good prompt.**
+
+## The Prompt
+
+<div style={{textAlign: 'left'}}>
+  <img src={Skippy} style={{width: "700px"}} />
+  <p style={{color: "gray", fontSize: "12px", fontStyle: "italic"}}>Full Skippy chat bot prompt from article header</p>
+</div>
+
+<a href="https://learnprompting.org/docs/basics/prompting">Prompting is the process of instructing an AI to do something. </a> As you have probably seen in ChatGPT examples online, you can prompt GPT-3 with just about anything. Common use cases are summarizing text, writing content based on a description, or creating things like poems, recipes, and much more.
+
+<p></p>
+
+**The core abstraction to this process is input/output.** You input information and then the model uses the data it was trained on to predict the best completion to your input. An important thing to note is that an LLM like GPT-3 isn't responding to a prompt, it is determining what tokens come next based on a probability distribution of tokens is has seen during pre-training(@jurafsky2009). Put simply, it is figuring out what words comes next given some previous words. 
```

Comment:
What does `The core abstraction to this process is input/output.` mean? Do you need this sentence?